tests/view_tests/tests/test_debug.py

```diff
@@ -177,7 +177,7 @@ def test_technical_404(self):
             html=True,
         )
         self.assertContains(response, "Raised by:", status_code=404)
-        self.assertContains(response, "view_tests.views.technical404", status_code=404)
+        self.assertContains(response, "<td>view_tests.views.technical404</td>", status_code=404)
```

Comment:
I don't think it makes a difference here, but the other tests use `html=True`

```suggestion
        self.assertContains(response, "<td>view_tests.views.technical404</td>", status_code=404, html=True)
```